.github/workflows/main.yml

```
@@ -86,166 +87,72 @@ jobs:
-          target: "/tmp/backend_deploy_temp_dir/"
-      - name: Deploy Backend on Server
+      - name: Deploy to Server via SSH
         uses: appleboy/ssh-action@master
         with:
           host: ${{ secrets.SERVER_HOST }}
           username: ${{ secrets.SSH_USERNAME }}
           key: ${{ secrets.SSH_PRIVATE_KEY }}
           script: |
-            bash -c '
-            echo "Iniciando despliegue de Backend en el servidor..."
+            bash -c ' 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)